### PR TITLE
Cloud changelog - Sept 28th release

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

**Review by agnivade — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 1]
```diff
@@ -4,12 +4,52 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Updated the post textbox measurement code to be more reliable.
+ - Pre-packaged Calls version v0.19.1.
+ - The ``invite`` slash command now supports custom user groups.
+ - Re-enabled the remote marketplace functionality (if configured as per ``PluginSettings.EnableRemoteMarketplace``).
+ - Added the ability to convert a Group Message channel to a private channel.
+ - Group Message channels now behave like Direct Message channels, showing a new mention on every message.
+
+#### Administration
+ - Added two new URL parameters to ``GET /api/v4/groups``: ``include_archived`` and ``filter_archived``. Added the ability to restore archived groups from the user groups modal.
+ - Moved the ``request`` package into the public shared folder.
```
Comment (isacikgoz) [inline source/install/cloud-changelog.md:32]: This might go to API changes.

**Review by isacikgoz — APPROVED** [review]
Looks good, just one minor change is required.

File: source/install/cloud-changelog.md [hunk 2]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+## Release 2023-09-28
+
+### Improvements
+
+#### User Interface (UI)
+ - GitLab plugin v1.7.0 is now available via prepackaged plugins.
+ - Added additional reaction options when viewing threads or messages when the sidebar is larger than its minimum width.
+ - Added block changes to name, display name, and purpose for direct and group messages.
+ - Added a link to notification documentation in the **Notification Settings** modal.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:22]:
```suggestion
 - Added a link to [notification documentation](https://docs.mattermost.com/preferences/manage-your-notifications.html) in the **Notification Settings** modal.
```

File: source/install/cloud-changelog.md [hunk 3]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
 - [Release 2023-08-03](#release-2023-08-03)
 - [Release 2023-07-26](#release-2023-07-26)
-- [Release 2023-07-20](#release-2023-07-20)
+
+## Release 2023-09-28
+
+### Improvements
+
+#### User Interface (UI)
+ - GitLab plugin v1.7.0 is now available via prepackaged plugins.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:19]: Suggestion: Document all prepackaged plugins consistently. Consider the Calls bullet point below as our starting point: ``Pre-packaged Calls version v0.19.1.``

In this case: ``Pre-packaged GitLab plugin version v1.7.0.``

File: source/install/cloud-changelog.md [hunk 4]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added additional reaction options when viewing threads or messages when the sidebar is larger than its minimum width.
+ - Added block changes to name, display name, and purpose for direct and group messages.
+ - Added a link to notification documentation in the **Notification Settings** modal.
+ - Updated the post textbox measurement code to be more reliable.
+ - Pre-packaged Calls version v0.19.1.
+ - The ``invite`` slash command now supports custom user groups.
+ - Re-enabled the remote marketplace functionality (if configured as per ``PluginSettings.EnableRemoteMarketplace``).
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:26]:
```suggestion
 - Re-enabled the remote marketplace functionality, when configured as per ``PluginSettings.EnableRemoteMarketplace`` [documentation](https://docs.mattermost.com/configure/plugins-configuration-settings.html#plugins-enableremotemarketplace).
```

File: source/install/cloud-changelog.md [hunk 5]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Added the ability to convert a Group Message channel to a private channel.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:27]:
```suggestion
 - Added the ability to convert a group message channel to a private channel.
```

File: source/install/cloud-changelog.md [hunk 6]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Group Message channels now behave like Direct Message channels, showing a new mention on every message.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:28]:
```suggestion
 - Group message channels now behave like direct message channels, showing a new mention for every new message.
```

File: source/install/cloud-changelog.md [hunk 7]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+
+#### Administration
+ - Added two new URL parameters to ``GET /api/v4/groups``: ``include_archived`` and ``filter_archived``. Added the ability to restore archived groups from the user groups modal.
```
Comment (cwarnermm) [inline source/install/cloud-changelog.md:31]:
```suggestion
 - Added 2 new URL parameters to ``GET /api/v4/groups``: ``include_archived`` and ``filter_archived``. Added the ability to restore archived groups from the user groups modal.
```

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline

**Review by streamer45 — APPROVED** [review]

File: source/install/cloud-changelog.md [hunk 8]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - Moved the ``request`` package into the public shared folder.
+
+### Bug Fixes
+ - Fixed an issue with keyboard support for some menus with submenus.
+ - Fixed an issue with a disappearing punctuation when following a group mention.
+ - Fixed an issue where compliance export jobs were not able to start after disabling and enabling the compliance export.
+ - Fixed a potential read after write issue when loading a license.
+
+### Known Issues
+ - Known issues related to the new feature to convert a Group Message channel to a private channel: [MM-54524](https://mattermost.atlassian.net/browse/MM-54524), [MM-54525](https://mattermost.atlassian.net/browse/MM-54525), [MM-54526](https://mattermost.atlassian.net/browse/MM-54526), [MM-54541](https://mattermost.atlassian.net/browse/MM-54541), [MM-54542](https://mattermost.atlassian.net/browse/MM-54542), [MM-54543](https://mattermost.atlassian.net/browse/MM-54543).
```
Comment (marianunez) [inline source/install/cloud-changelog.md:52]: Seems https://mattermost.atlassian.net/browse/MM-54524 may not have been an issue to this feature so we can probably remove from here? and https://mattermost.atlassian.net/browse/MM-54543 we've closed as Won't Fix and is a mobile change that won't go out in this Cloud release so we can probably remove as well.

File: source/install/cloud-changelog.md [hunk 9]
```diff
@@ -4,12 +4,54 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
...
+ - ``/invite`` user to a channel incorrectly posts an ephemeral message of ``<no value> added to {channel_name} channel`` [MM-54555](https://mattermost.atlassian.net/browse/MM-54555).
+ - Left-hand side resize option overrides the **Browse/Create Channel** menu [MM-54367](https://mattermost.atlassian.net/browse/MM-54367).
```
Comment (marianunez) [inline source/install/cloud-changelog.md:54]: This one is only an issue if you have the To-Do plugin installed which I'm not sure you can install in Cloud but we can at least mention that in any case: 
```suggestion
 - Left-hand side resize option overrides the **Browse/Create Channel** menu if To-Do plugin is installed [MM-54367](https://mattermost.atlassian.net/browse/MM-54367).
```

**Review by marianunez — APPROVED** [review]

**Pull request merged**